rasa/core/tracker_store.py

```diff
@@ -1664,3 +1657,21 @@ async def retrieve_full_tracker(
             if isawaitable(result)
             else result  # type: ignore[return-value]
         )
+
+
+class TrackerEventDiffEngine:
```

Comment:
I wonder if this should not actually be moved to the trackers module instead? wdyt?

Reply:
That actually makes more sense 👍🏾